ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -196,6 +196,60 @@ periodics:
           sources:
           - secret:
               name: cluster-secrets-azure
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: periodic-ci-azure-etcdbackuprecovery-e2e
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --git-ref=openshift/openshift-azure@master
```

Comment:
Use your own branch until your PR gets merged, then we can update to openshift/openshift-azure@master.

Reply:
This should be through `extra_refs` in the ProwJob config, not the flag

Reply:
Do you have an example with `extra_refs` and `ci-operator`? I couldn't find any and I don't think it works at the moment, hence https://github.com/openshift/ci-operator/issues/142.